base/iterators.jl

```diff
@@ -1095,4 +1094,40 @@ eltype(::Type{Stateful{T, VS}} where VS) where {T} = eltype(T)
 IteratorEltype(::Type{Stateful{T,VS}}) where {T,VS} = IteratorEltype(T)
 length(s::Stateful) = length(s.itr) - s.taken
 
+"""
+    only(x)
+
+Returns the one and only element of collection `x`, and throws an `ArgumentError` if the
+collection has zero or multiple elements.
```

Comment:
```suggestion
collection has zero or multiple elements.
See also [`first`](@ref), [`last`](@ref).
```

Reply:
goo idea! thanks :)
done

Reply:
(added offline so I could be sure to exactly match the syntax/style from the `Pairs` docstring above) 💛